### DISCUSSION TITLE
Execute ECS task from Dagster local

### DISCUSSION THREAD

**Discussion by jsanjay63** [dagster-io/dagster#14159] [Q&A]
I am relatively new to Dagster. I am trying to execute an ECS task from dagster using this standalone python file as a job:

```
from dagster import job, op

@op()
def my_op(context):
  context.log.info('running')

@job(
  tags={
    "ecs/task_definition": "arn_task_definition",
    "ecs/container_name": "container_name"
  }
)
def my_job():
  my_op()
```
Command to execute:
```
dagster dev -f hello_dag.py
```
In the logs, I am able to get "running". But, I couldn't find any logs related to ECS. I also checked ECS console, but there are no task triggered. Can someone please help with executing the ECS task? Thanks

**Comment by smackesey** [comment]
Hi @jsanjay63, have you followed this full guide: https://docs.dagster.io/deployment/guides/aws

**Reply by jsanjay63** [reply]
Not exactly. I am trying to execute from my local environment. I'd like to trigger an ECS task from any machine.
 In production, I'll try to execute in AWS. But, even in that case, it maybe from other ECS cluster.
Could you please help with executing from local environment? Thanks

**Reply by jzgrzebnicki** [reply]
Were you able to do so?
